package.json

```diff
@@ -6,8 +6,11 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
```

Comment:
- [ ] You have installed react-redux, but unfortunately, I miss seeing the folders and files where redux logic will be placed. This is a requirement of the project, so let's please make sure to include them.

![image](https://user-images.githubusercontent.com/79114233/144951120-67d9f994-5abb-47b4-b021-591eb242811a.png)

Reply:
fixed